Tensile/KernelWriterAssembly.py

```diff
@@ -10623,6 +10626,8 @@ def globalWriteElements(self, kernel, vectorWidths, elements,
     # End label
     kStr += "label_%s:%s"%(endLabel, self.endLine)
     self.vgprPool.checkIn(tmpVgpr)
+    if bf16CVTVgpr != None:
```

Comment:
Should probably use 
if bf16CVTVgpr is not None

Reply:
update